### DISCUSSION TITLE
Examples/Quick Reference

### DISCUSSION THREAD

**Discussion by tjdwill** [hylang/hy#2568] [General]
Hello, I have been playing with Hy for a bit now, and I think it would be useful for those of us who aren't really as familiar with Lisp to have some examples and/or a quick reference for learning the ropes.

I've placed a sample of a reference I've been writing for myself at the [following page](https://github.com/tjdwill/HylangPractice/tree/main), and I would love to know if this something along these lines could be useful to others.

Also, just want to say that this project is really cool, and I can tell a lot of thought went into it.

**Comment by Kodiologist** [comment]
Have you seen the tutorial in the manual? It covers largely the same topics. You may want to add to it.

**Reply by tjdwill** [reply]
I have, and while it was useful in getting started, I did the above for myself to have more in-depth examples. I'll read up on how to add to the docs.
